packages/client/src/recommendations/types/getProductRecommendations.types.ts

```diff
@@ -0,0 +1,50 @@
+import type { Config } from '../../types';
+
+export type GetProductRecommendations = (
+  query: GetProductRecommendationQuery,
+  config?: Config,
+) => Promise<GetProductRecommendation[]>;
+
+export type GetProductRecommendationQuery = {
+  productId: number;
+  strategyName: string;
+  howMany?: number;
+  countryCode?: string;
+  gender?: string;
+  userType?: string;
+  userIdentifier?: string;
+  categoryId?: number;
+  brandId?: string;
+  merchantId?: number;
+};
+
+export type GetProductRecommendation = {
```

Comment:
This should be called ProductRecommendation as it is independent from the request method.